apps/call.go

```diff
@@ -134,29 +133,29 @@ type CallResponse struct {
 // Apps will use the CallResponse struct to respond to a CallRequest, and the proxy will
 // decorate the response using the ProxyCallResponse to provide additional information.
 type ProxyCallResponse struct {
-	*CallResponse
+	CallResponse
 
 	// Used to provide info about the App to client, e.g. the bot user id
 	AppMetadata *AppMetadataForClient `json:"app_metadata"`
 }
 
-func NewProxyCallResponse(response *CallResponse, metadata *AppMetadataForClient) *ProxyCallResponse {
-	return &ProxyCallResponse{
+func NewProxyCallResponse(response CallResponse, metadata *AppMetadataForClient) ProxyCallResponse {
+	return ProxyCallResponse{
 		response,
 		metadata,
 	}
 }
 
-func NewErrorCallResponse(err error) *CallResponse {
-	return &CallResponse{
+func NewErrorCallResponse(err error) CallResponse {
+	return CallResponse{
 		Type: CallResponseTypeError,
 		// TODO <>/<> ticket use MD instead of ErrorText
```

Comment:
The comment is still relevant, the field is now `Markdown`. The idea was, we already have the CallType=error, and if so `Markdown` can (and should?) be used to send the error's text, rather than a special field in the struct. The train has probably already left the station though... maybe too late to change.

Not relevant to this PR, right?

Reply:
Got it now, thanks for the explanation :+1: Surly it's non blocking for this PR.

1/5 that this seems like a good simplification and we should file a ticket. @larkox What do you think?

Reply:
Whether we can output markdown or not, and what type of markdown is potentially location based. I am OK with this simplification, but because of this reason, I would change the name from markdown to something more abstract like output.